src/models/github-server-app.ts

```diff
@@ -107,16 +107,16 @@ export class GitHubServerApp extends Model {
 		return { jiraHost };
 	}
 
-	static async getForGitHubServerAppId(
-		gitHubServerAppId: number
+	static async getByIdPk(
```

Comment:
/nit
Isn't this provided by the sequelize already?

Reply:
pretty sure it is provided, but don't to remove this now